src/components/About/About.jsx

```diff
@@ -47,101 +48,75 @@ function About() {
+                  href="https://discordapp.com/invite/Terasology"
+                >
+                  Discord
+                </a>
+                .
+              </p>
+            </HighlightBox>
           </Col>
           <Col md="12">
-            <div
-              className="with-bg-image"
-              style={{ backgroundImage: `url(${withPrefix("/images/9.jpg")})` }}
+            <HighlightBox
+              title="Modding API"
+              bgImage={withPrefix("/images/9.jpg")}
+              align="end"
             >
-              <div className="right-overlay_about_content">
-                <Row>
-                  <Col className="empty-about-content" md="7" />
-                  <Col md="5" className="text-white float-right ">
-                    <div className="ml-2 mr-4 ">
-                      <h3 className="ml-2 mb-3 text-white">Modding API</h3>
-                      <p className="ml-2 mr-3 text-justify ">
-                        {`Terasology's`} engine uses a whitelisting approach to
-                        expose an API for modules using two primary methods and
-                        a rarely needed third one:
-                      </p>
-                      <ul>
-                        <li>
-                          Classes or packages marked with the @API annotation
-                        </li>
-                        <li>
-                          Classes or packages in the basic whitelist defined in
-                          ExternalApiWhitelist.java
-                        </li>
-                        <li>
-                          Rarely blocks of code in the engine may be hit in a
-                          way requiring use of
-                          AccessController.doPrivileged(...) usually module
-                          authors do not need to worry about this but once in a
-                          while it could explain something quirky
-                        </li>
-                      </ul>
-                    </div>
-                  </Col>
-                </Row>
-              </div>
-            </div>
+              <p>
+                {`Terasology's`} engine uses a whitelisting approach to expose
+                an API for modules using two primary methods and a rarely needed
+                third one:
+              </p>
+              <ul className="text-left">
+                <li>Classes or packages marked with the @API annotation</li>
+                <li>
+                  Classes or packages in the basic whitelist defined in
+                  ExternalApiWhitelist.java
+                </li>
+                <li>
+                  Rarely blocks of code in the engine may be hit in a way
+                  requiring use of AccessController.doPrivileged(...) usually
+                  module authors do not need to worry about this but once in a
+                  while it could explain something quirky
+                </li>
+              </ul>
```

Comment:
I did not go down the rabbit hole of figuring out why exactly the font size of the list items is different from the font size of the other text in paragraphs.

I looks the same as before, and I think we should look at our CSS styling once we cleaned up components and are making better reuse of components.